splunk-otel-android/src/main/java/com/splunk/rum/NetworkMonitor.java

```diff
@@ -78,19 +77,8 @@ public void onAvailable(boolean deviceIsOnline, CurrentNetwork activeNetwork) {
                 Span available =
                         tracer.spanBuilder("network.change")
                                 .setAttribute(NETWORK_STATUS_KEY, "available")
-                                .startSpan()
-                                // put these after span start to override what might be set in the
-                                // RumAttributeAppender.
```

Comment:
I'd leave the comment -- it does provide some additional explanation for why the `appendNetworkAttributes()` does not accept a `SpanBuilder` (or some kind of `setAttribute()` lambda interface, common for both span and builder)